models/clm/tutorial/README.rst

```diff
@@ -31,7 +31,7 @@ for CLM-specific assistance.
 
 If you are new to DART, we recommend that you become familiar
 by first working through the :doc:`../../../theory/readme` and then
-understanding the :doc:`DART getting started <../../../README>` documentation.
+understanding the :ref:`DART getting started <Welcome page>` documentation.
```

Comment:
```suggestion
understanding the :ref:`Getting Started <Welcome page>` documentation.
```

Reply:
Same comment: I think this is to be clear that a user should read the DART getting started documentation (rather than just WRF).